Adds Files page callout in Query. (#92)
web3-storage/docs: docs/how-tos/query.md

For more details about the fields in this JSON response, including the format of the `pins` and `deals` objects, see the [JavaScript client library reference][reference-js-status].

> :::tip Use the website

Reviewer: ```suggestion
:::tip
```
So as not to imply that using the website is the only option.

> The **Files** page on [web3.storage](https://web3.storage) lists some commonly used attributes found using `query()`.

Reviewer: ```suggestion
If you're looking for info on files you've uploaded, you can also use the **Files** page on [web3.storage](https://web3.storage) to see the values for some of the more commonly-used attributes returned by `query()`, namely `created`, `cid`, `dagSize`, and the `status` and `deals` objects of `pins`.
```
Clarify that this only works for files _you've_ uploaded, and add the list of values.

> ![Web3.Storage showing the Files page](./images/files-page-in-web3-storage.png)

Reviewer: ```suggestion
```
Suggest we omit this for now for two reasons: 
- screenshots in tip boxes are confusing visually
- this screenshot will change in the coming days

**If you agree, please remove the image from the directory as well.**
